docs/sources/setup-grafana/installation/docker/index.md

```diff
@@ -8,253 +8,318 @@ weight: 200
+         restart: unless-stopped
+         ports:
+           - '3000:3000'
+   ```
 
-```bash
-cd packaging/docker/custom
-docker build \
-  --build-arg "GRAFANA_VERSION=latest" \
-  --build-arg "GF_INSTALL_IMAGE_RENDERER_PLUGIN=true" \
-  -t grafana-custom .
+1. To run `docker-compose.yaml`, run the following command:
 
-docker run -d -p 3000:3000 --name=grafana grafana-custom
-```
+   ```bash
+   # first go into the directory where you have created this docker-compose.yaml file
+   cd /path/to/docker-compose-folder 
 
-## Migrate from previous Docker containers versions
+   # then start the grafana container 
+   docker compose up -d docker-compose.yaml
+   ```
 
-This section contains important information if you want to migrate from previous Grafana container versions to a more current one.
+   Where:
 
-### Migrate to v7.3 or later
+   d = detached mode \
+   up = to bring the container up and running
 
-The Grafana Docker image runs with the `root` group (id 0) instead of the `grafana` group (id 472), for better compatibility with OpenShift. If you extend the official Docker image you may need to change your scripts to use the `root` group instead of `grafana`.
+To determine that Grafana is running, open a browser window and type `IP_ADDRESS:3000`. The sign in screen should appear.
 
-### Migrate to v6.5 or later
+### Stop the Grafana container
 
-Grafana Docker image now comes in two variants, one [Alpine](http://alpinelinux.org) based and one [Ubuntu](https://ubuntu.com/) based, see [Image Variants](#image-variants) for details.
+To stop the Grafana container, run the following command:
 
-### Migrate to v6.4 or later
+```bash
+docker compose down
+```
 
-Grafana Docker image was changed to be based on [Alpine](http://alpinelinux.org) instead of [Ubuntu](https://ubuntu.com/).
+> **Note:** For more information about using Docker Compose commands, refer to [docker compose](https://docs.docker.com/engine/reference/commandline/compose/).
 
-### Migrate to v5.1 or later
+### Save your Grafana data
 
-The Docker container for Grafana has seen a major rewrite for 5.1.
+When you use Docker containers, the data inside them is temporary by default. This means that if you don't specify where to store the information, all the Grafana data will be lost when you stop the Docker container. To avoid losing your data, you can set up [persistent storage](https://docs.docker.com/storage/volumes/) or [bind mounts](https://docs.docker.com/storage/bind-mounts/) for your container.
 
-**Important changes**
+#### Use persistent storage (recommended)
 
-- File ownership is no longer modified during startup with `chown`.
-- Default user ID is now `472` instead of `104`.
-- Removed the following implicit volumes:
-  - `/var/lib/grafana`
-  - `/etc/grafana`
-  - `/var/log/grafana`
+[Need intro here on why you'd use this option over bind mounts. Why is this recommended?]
 
-#### Removal of implicit volumes
+Complete the following steps to use persistent storage.
```

Comment:
@usmangt , These are the same as the steps documented in ### Run the latest stable version of Grafana. Do we need them in both places?

Reply:
Same as above

Reply:
Ok, let's do this. After you've accepted my changes, I'll pull down the branch and play around with the structure. I'd like to have just a single section for this content.

Reply:
Decided to leave the structure as is because there are differences when using docker compose.